dev-qt/qtcore/qtcore-5.15.2-r10.ebuild

```diff
@@ -0,0 +1,104 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+KDE_ORG_COMMIT=7e3f3b0c1f7f417e52ca374228f5afb824a913d6
+QT5_MODULE="qtbase"
+inherit linux-info qt5-build
+
+DESCRIPTION="Cross-platform application development framework"
+SLOT=5/$(ver_cut 1-3)
+
+if [[ ${QT5_BUILD_TYPE} == release ]]; then
+	KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~ppc ~ppc64 ~riscv ~sparc ~x86"
+fi
+
+IUSE="icu old-kernel systemd"
+
+DEPEND="
+	dev-libs/double-conversion:=
+	dev-libs/glib:2
+	dev-libs/libpcre2[pcre16,unicode]
```

Comment:
```suggestion
	dev-libs/libpcre2:=[pcre16,unicode]
```

Reply:
the comment in `libpcre2-10.37-r2.ebuild` says "libpcre2-posix.so version", but libQt5Core.so links against libpcre2-16.so, not libpcre2-posix.so, so I'm not sure about this.

Reply:
Ooh, I see. You're right. I'll add this to metadata.xml in pcre2.

Reply:
Done as https://github.com/gentoo/gentoo/commit/1776ff0260ce640f141e7b4f5345054c890109ab. Cheers!